app/views/shared/_asia_fields.html.slim

```diff
@@ -17,22 +17,22 @@
               tr
                 th scope="row"  Sensory
                 td 
-                  .form-control.input
+                  = f.input_field :neurological_sensory_level_left, label: false, collection: Asia.level_of_injuries.keys
                 td 
-                  .form-control.input
+                  = f.input_field :neurological_sensory_level_right, label: false, collection: Asia.level_of_injuries.keys
               tr
                 th scope="row"  Motor
                 td 
-                  .form-control.input
+                  = f.input_field :neurological_motor_level_left, label: false, collection: Asia.level_of_injuries.keys
                 td 
-                  .form-control.input
+                  = f.input_field :neurological_motor_level_right, label: false, collection: Asia.level_of_injuries.keys
       .col-md-4
         label ASIA Impairment Scale
-        = f.input :asia_score, :prompt => "Select one", collection: Patient.asia_levels.keys, :label => false, input_html: { class: "form-control" }
+        .form-control.input Missing JS auto-calc
         hr
 
         label Neurological Level of Injury
-        = f.input :level_of_injury, :label => "Neurological Level of Injury", :prompt => "Select one", collection: Patient.level_of_injuries.keys, :label => false, :no_col => true
+        .form-control.input Missing JS auto-calc
```

Comment:
Yep...note for later.  The Asia Classification is an algorihm based off the other scores. You shouldn't have to type it in.

The level of injury is basically find the "highest" value of given 8 measurements.